### ISSUE TITLE
Auto Update

### ISSUE THREAD

**Issue by wbuk** [marxjohnson/moodle-block_accessibility#122]
A brilliant block, perfect!

Please would it be possible for Accessibility to use the Auto Update Notification feature of Moodle 3.5, Everytime we upgrade moodle, it days missing from Disk when the other extensions are automatically detected and then installed.    It would save a lot of time.  Thank you

Also, if you are considering an update, we use it for an education site for school children to learn.  Schools now have requested more overlay colours then the block supports.  On your next update can you add room for a further 3.  It would be very much appreciated.

Thank you.

**Comment by wbuk** [comment]
Hi

I have just updated moodle and accessibility is still not auto-updating.  Is there any chance someone can make it use the auto-update function of moodle, it would be very much appreciated.  Aweseom block, great for helping special needs students, our special needs students could not do without it.

Thank you so much.
